crates/load-test/src/main.rs

```diff
@@ -0,0 +1,450 @@
+            StarkHash::from_hex_str(
+                "0x58d8604f22510af5b120d1204ebf25292a79bfb09c4882c2e456abc2763d4a",
+            )
+            .unwrap(),
+        )),
+    )
+    .await?;
+    Ok(())
+}
+
+async fn task_block_transaction_count_by_number(user: &mut GooseUser) -> TransactionResult {
+    get_block_transaction_count_by_number(
+        user,
+        BlockNumberOrTag::Number(StarknetBlockNumber(1000)),
+    )
+    .await?;
+    Ok(())
+}
+
+async fn task_transaction_by_hash(user: &mut GooseUser) -> TransactionResult {
+    get_transaction_by_hash(
+        user,
+        StarknetTransactionHash(
+            StarkHash::from_hex_str(
+                "0x39ee26a0251338f1ef96b66c0ffacbc7a41f36bd465055e39621673ff10fb60",
+            )
+            .unwrap(),
+        ),
+    )
+    .await?;
+    Ok(())
+}
+
+async fn task_transaction_by_block_number_and_index(user: &mut GooseUser) -> TransactionResult {
+    get_transaction_by_block_number_and_index(
+        user,
+        StarknetBlockNumber(1000),
+        StarknetTransactionIndex(3),
+    )
+    .await?;
+    Ok(())
+}
+
+async fn task_transaction_by_block_hash_and_index(user: &mut GooseUser) -> TransactionResult {
+    get_transaction_by_block_hash_and_index(
+        user,
+        StarknetBlockHash(
+            StarkHash::from_hex_str(
+                "0x58d8604f22510af5b120d1204ebf25292a79bfb09c4882c2e456abc2763d4a",
+            )
+            .unwrap(),
+        ),
+        StarknetTransactionIndex(3),
+    )
+    .await?;
+    Ok(())
+}
+
+async fn task_transaction_receipt_by_hash(user: &mut GooseUser) -> TransactionResult {
+    get_transaction_receipt_by_hash(
+        user,
+        StarknetTransactionHash(
+            StarkHash::from_hex_str(
+                "0x39ee26a0251338f1ef96b66c0ffacbc7a41f36bd465055e39621673ff10fb60",
+            )
+            .unwrap(),
+        ),
+    )
+    .await?;
+    Ok(())
+}
+
+async fn task_block_number(user: &mut GooseUser) -> TransactionResult {
+    block_number(user).await?;
+    Ok(())
+}
+
+async fn task_syncing(user: &mut GooseUser) -> TransactionResult {
+    syncing(user).await?;
+    Ok(())
+}
+
+async fn task_call(user: &mut GooseUser) -> TransactionResult {
+    call(
+        user,
+        ContractAddress(
+            StarkHash::from_hex_str(
+                "0x06ee3440b08a9c805305449ec7f7003f27e9f7e287b83610952ec36bdc5a6bae",
+            )
+            .unwrap(),
+        ),
+        &[
+            "0x01e2cd4b3588e8f6f9c4e89fb0e293bf92018c96d7a93ee367d29a284223b6ff",
+            "0x071d1e9d188c784a0bde95c1d508877a0d93e9102b37213d1e13f3ebc54a7751",
+        ],
+        "0x3d7905601c217734671143d457f0db37f7f8883112abd34b92c4abfeafde0c3",
+        BlockHashOrTag::Hash(StarknetBlockHash(
+            StarkHash::from_hex_str(
+                "0x47c3637b57c2b079b93c61539950c17e868a28f46cdef28f88521067f21e943",
+            )
+            .unwrap(),
+        )),
+    )
+    .await?;
+    Ok(())
+}
```

Comment:
Could you add some comments on what StarkNet stuff this is executing?

Reply:
Sure. I certainly don't know the details, but this is basically a test contract deployed in block 0.

Reply:
Okay great, yeah that's enough info already -- just means we don't have to go crawl block explorers for the hashes etc

Reply:
Added all I know as comments.